pkgs/top-level/perl-packages.nix

```
@@ -1521,6 +1521,27 @@ let
     };
   };
 
+  BioPerl = buildPerlPackage {
```

Consider 
```suggestion
  BioPerl = buildPerlModule {
```

so you can eliminate the manual dependency for `ModuleBuild` below.

---

Many thanks for your helpful review ! The package does not have a Build.PL so `buildPerlModule `cannot be used. But `ModuleBuild` is still needed as a dependency, otherwise it will not compile.

---

Ah right, its built with Dist::Zilla - a bit odd to use Module::Build here, but that's an upstream thing 😸